apps/web/src/hooks/useFiles.ts

```diff
@@ -20,7 +20,9 @@ export function useFiles() {
     queryFn: async () => fetchFiles(`@zk-kit/${selectedProject}-artifacts@${selectedVersion}`),
     select: data => {
       const regex = new RegExp(`^packages/${selectedProject}/.*\\.(${artifacts.join('|')})$`)
-      return data.filter(file => regex.test(file)).map(file => file.replace(`packages/${selectedProject}/`, ''))
+      return data.filter(file => regex.test(file)).filter(file => !file.endsWith('package.json')).map(file =>
```

Comment:
it would probably be more efficient to edit the regex instead.
But I am bad at regexes, and find them hard to read. So using a second more straightforward `.filter` instead